include/psa/crypto_config.h

```diff
@@ -80,6 +80,20 @@
 #define PSA_WANT_ALG_TLS12_PSK_TO_MS            1
 #define PSA_WANT_ALG_XTS                        1
 
+#define PSA_WANT_ECC_BP_R1_256                  1
```

Comment:
I guess this is for curves belonging to the Brainpool P family. Thus to align with the PSA spec this should rather be:
PSA_WANT_ECC_BRAINPOOL_P_R1_256. Please change wherever applicable.